build/NuGetAdditionalFiles/RunCsc

```diff
@@ -1,5 +1,15 @@
 #!/bin/sh
 
+set -e
+set -u
+
 THISDIR=$(dirname $0)
 
-dotnet $THISDIR/csc.dll "$@"
+# The expression ${VAR:+TEXT} expands to:
+#  If VAR is null or empty => empty string
+#  If VAR is non-empty => TEXT
+# Do this expression to make sure HOST_PATH ends with a slash,
+#  but if it's unset, have it be completely empty (look up dotnet in PATH)
+HOST_PATH=${DOTNET_HOST_PATH:+${DOTNET_HOST_PATH-}/}
```

Comment:
Why would we do this vs. the approach in the CMD files: if `DOTNET_HOST_PATH` is defined then just use it directly?